services/dumper.js

```diff
@@ -76,6 +76,13 @@ function Dumper(config) {
     return _.kebabCase(table);
   }
 
+  function getModelName(table) {
+    if (table === 'sessions') {
+      return 'session';
+    }
```

Comment:
It does not look to be a long term solution against this kind of route conflicts issue.

Reply:
What solution do you suggest ?